test/out/updater/__snapshots__/nsisUpdaterTest.js.snap

```diff
@@ -196,7 +196,7 @@ Object {
       "url": "TestApp-Setup-1.1.0.exe",
     },
   ],
-  "releaseName": "v1.5.2-beta.3",
```

Comment:
@develar actually, the easiest way to see why this is broken is to look here at the changes I had to make in the snapshot file here, and at Line 213. At present your snapshot doesn't match the state of the releases in your test project.

Reply:
That is, per https://github.com/develar/__test_nsis_release/releases.atom, there is no release with empty release notes, a releaseName of `v1.5.2-beta.3`, and a tag of `v1.1.0`.